src/apps/statements/repo/modelsRepo/utils/mongoModels/constants.ts

```diff
@@ -1,3 +1,11 @@
+import getNumberOption from 'jscommons/dist/config/getNumberOption';
+
 export const STATEMENTS_COLLECTION_NAME = 'statements';
 export const LRS_COLLECTION_NAME = 'lrs';
 export const FULL_ACTIVITIES_COLLECTION_NAME = 'fullActivities';
+
+const WRITE_CONCERN_DEFAULT = 3;
```

Comment:
MongoDB's default is `majority` and here our default is 3. This would create behavioural change immediately upon deployment.